DQM/L1TMonitor/interface/L1MUTK.h

```diff
@@ -0,0 +1,94 @@
+#ifndef L1MUTK_H
```

Comment:
- rename the file (and the class to something more intuitive) Phase2L1TkMuonMonitor or just L1TkMuonMonitor
- Use essentially full file name: DQM_L1TMonitor_L1TkMuonMonitor for the include guard (ifndef ... part)